index.html

```diff
@@ -28,12 +28,200 @@ <h1>I'm David <br>Glad to see you!</h1>
+                </li>
+                <li class="lang">
+
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="js">javaScript</li>
+                    </ul>
+
+                </li>
+                <li class="view">
+                    <a href="">
+                        <h3>See Project</h3>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="projects">
+            <img class="img" src="./images/Snapshoot Portfolio (3).png">
+
+            </img>
+            <ul class="projectInfo">
+                <li class="mps">
+                    <h3>Multi-Post <br> Stories</h3>
+                </li>
+                <li class="techstack">
+                    <div>
+                        <ul>
+                            <li class="canopy">CANOPY</li>
+                            <li class="bed">Back End Dev</li>
+                            <li class="year">2015</li>
+                        </ul>
+                    </div>
+                </li>
+                <li class="aboutProject">
+                    A daily selection of privately <br> personalized reads: no accounts or <br> sign-ups required.
+                </li>
+                <li class="lang">
+
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="js">javaScript</li>
+                    </ul>
+
+                </li>
+                <li class="view">
+                    <a href="">
+                        <h3>See Project</h3>
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="aboutme">
+        <div class="aboutInfo">
+            <h2>About <br> Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+                <h3>
+                    LET'S CONNECT
+                </h3>
+                <ul class="icons">
+                    <li><a href="./index.html"><img src="./images/Vector.svg" alt="twitter icon"></a></li>
+                    <li><a href="./index.html"><img src="./images/Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                    <li><a href="./index.html"><img src="./images/Vector (1).svg" alt=""></a></li>
+                    <li><a href="./index.html"><img src="./images/Vector (2).svg" alt="github icon"></a></li>
+                    <li><a href="./index.html"><img src="./images/Vector (3).svg" alt=""></a></li>
+                </ul>
+                <h3 class="get">
+                    Get my resume
+                </h3>
```

Comment:
- [ ] Dear @David-Lanzz, Please make the `Get my resume` a button or a link as it will be used to link to your resume later. and  follow the button styling: `Active`, `Hover`, and `Pressed`
![Screenshot 2023-01-13 210721](https://user-images.githubusercontent.com/69077061/212389679-789a266d-7524-49d3-810a-2fe0b63042e4.png)